content/en/tracing/trace_collection/custom_instrumentation/_index.md

```diff
@@ -29,23 +14,38 @@ algolia:
   tags: ['apm custom instrumentation']
 ---
 
-Custom instrumentation allows programmatic creation, modification, or deletion of traces to send to Datadog. This is useful for tracing in-house code not captured by automatic instrumentation, removing unwanted spans from traces, and for providing deeper visibility and context into spans, including adding any desired [span tags][1].
+## Overview
 
-Before instrumenting your application, review Datadog's [APM Terminology][2] and familiarize yourself with the core concepts of Datadog APM.
+Custom instrumentation allows for precise monitoring of specific components in your application. It allows you to capture observability data from in-house code or complex functions that aren't captured by automatic instrumentation. Automatic instrumentation includes [local library injection][6] or [single step instrumentation][5].
 
-If you use an open standard to instrument your code, see [Instrumenting with OpenTracing][3] or [Instrumenting with OpenTelemetry][4].
+Custom instrumentation involves embedding tracing code directly into your application code. This allows for the programmatic creation, modification, or deletion of traces to send to Datadog. 
 
-{{< partial name="apm/apm-manual-instrumentation.html" >}}
+## Getting started
 
+Follow the relevant documentation for your custom instrumentation approach to learn more:
 
-<br>
+- [Datadog libraries][2]
+- [OpenTelemetry][3]
+- [OpenTracing][4]
```

Comment:
Let's put these in the same order as in the nav:
- OTel
- Datadog
- OpenTracing - and maybe add a "this project is not supported anymore but might work for you if no other method works" (or maybe remove it from this list?)

Reply:
Great feedback--thanks, @kayayarai ! This should all be fixed, in addition to some suggestions from Irene and Vidur.